packages/core/styles/global.css

```diff
@@ -21,59 +20,44 @@ main {
     padding: 0;
     border: 0;
     overflow: hidden;
-    scrollbar-color:  var(--borders-light-grey) var(--accent-dark)
+    scrollbar-color:  var(--medium-grey) var(--accent-dark)
```

Comment:
Just noticed this doesn't seem to get applied for me on desktop
<img width="1518" alt="image" src="https://github.com/AllenInstitute/aics-fms-file-explorer-app/assets/24441706/1bab108d-263a-4c2f-a860-a800584a82f4">

Reply:
Having trouble replicating this - created an issue to check for styling differences on desktop since I there is at least 1-2 other things